api/hypershift/v1beta1/nodepool_conditions.go

```diff
@@ -18,6 +18,10 @@ const (
 	// NodePoolValidTuningConfigConditionType signals if the content within nodePool.spec.tuningConfig is valid.
 	// A failure here is unlikely to resolve without the changing user input.
 	NodePoolValidTuningConfigConditionType = "ValidTuningConfig"
+	// NodePoolSupportedVersionSkewConditionType signals if the NodePool points to a version that falls within the supported skew policy with the HostedCluster.
+	// When false, the NodePool will keep trying to operate as usual even though there are no guarantees
+	// A failure here is unlikely to resolve without changing the version input.
```

Comment:
changing the version input.
Can we be specific, e.g. without changing spec.release.
Can we also specify what is the supported skew policy.